src/styles/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ButtonBase, withStyles } from '@material-ui/core';
+import { styled } from '@mui/material';
+
+export const EndButtonBase = withStyles({
+  root: {
+    margin: 5,
+    color: 'fff',
+    backgroundColor: '#1976d1',
+    padding: '8px 90px',
+    borderRadius: '5px',
+    fontWeight: 'bold',
+    '&:focus': {
+      outline: 'auto',
+    },
+  },
+})(ButtonBase);
```

Comment:
Second thing, let's move away from Material's components. We want to use the native HTML `<button>` and apply the styles to it.

Reply:
@murilovmachado That means I should implement the styling in a CSS file under a class, right?

Reply:
@murilovmachado It's done.